src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -652,4 +652,18 @@
   <data name="CloseTabsAfterDefaultCommandKey" xml:space="preserve">
     <value>Close all tabs after the current tab</value>
   </data>
+  <data name="InvalidLinkContent" xml:space="preserve">
+    <value>The URI "{0}" is invalid. </value>

```

Comment:
```suggestion
    <value>The URI "{0}" is invalid.</value>
```

(trailing space)

Reply:
May be a little too technical. "There was an error reading this hyperlink." may be sufficient.